src/components/subscriptions/data/hooks.js

```diff
@@ -120,7 +120,7 @@ export const useActiveSubscriptionUsers = ({
     setLoading(true);
     LicenseManagerApiService.fetchSubscriptionUsers(
       subscriptionUUID,
-      { active_only: 1, page_size: 10000 },
+      { active_only: 1, page_size: 3 },
```

Comment:
you mentioned doing pagination on the backend, so I'm just wondering why are we passing in a page size here? So we still determine what the size should be on the frontend, pass it to the backend, and thus have the backend pre-limit its results to the amount we specify on the frontend, but is there a reason to specify it on the frontend?

Also, there's several other spots I see where we pass page_size to this method, but with the `PAGE_SIZE` constant which is a different amount (5) and I'm wondering if 1. if these are supposed to be the same number and 2. if so, we should define PAGE_SIZE in an external constants file so that this hook and the `AddLearnersStep.jsx` file where it's defined can both use the same constant/value.

Reply:
Oh, hahaha yeah, I'm actually not using this hook at all anymore; I can remove it.

Reply:
And the PAGE_SIZE constant should be 25. I don't actually have 25 users so I set it lower for testing and forgot to set it back. Thanks!

Reply:
Ah. Okay so that explains the hook and I guess the variable only needs to be in that one file, but still why are we defining page size on the frontend? Do we expect to not always use a static size (I haven't looked at the backend PR in a bit though so apologies if this is redundant)?